src/services/FeedProcessor.ts

```diff
@@ -181,7 +181,7 @@ class FeedProcessor {
    * @return multiInputs will be aggregated by the respective processor to be fetched in one API call
    */
   private separateInputs(uniqueFeedFetcherMap: {[hash: string]: FeedFetcher}) {
-    const multiFetchingInputsNames = ['CryptoComparePrice', 'CoingeckoPrice', 'UniswapV3'];
+    const multiFetchingInputsNames = ['CryptoComparePrice', 'CoingeckoPrice', 'UniswapV3', 'SovrynPriceFetcher'];
```

Comment:
I think you will need to rebase and use the FetcherName enum here

Reply:
Thanks for that